configure.ac

```diff
@@ -2480,7 +2481,8 @@ case $ac_sys_system/$ac_sys_release in
     }
     ]])],[ac_osx_32bit=yes],[ac_osx_32bit=no],[ac_osx_32bit=yes])
 
-    if test "${ac_osx_32bit}" = "yes"; then
+    if test -z "${MACOSX_DEFAULT_ARCH}"; then
```

Comment:
Why add this test?  I guess you are setting this variable when cross building.

Reply:
On a second look, what is `LIBTOOL_CRUFT` doing? Is it necessary?

Reply:
I haven't looked at this part of the build in a long time, so I don't know. I'm fairly sure that the configure script can be cleaned up a lot, even just the macOS related parts, but that should be out of scope for this PR.

Reply:
Use of `LIBTOOL_CRUFT` was removed in https://github.com/python/cpython/commit/250324952e63a51fe885d457d207082f249bbefd. I'll send a separate PR.